src/test/java/org/folio/oaipmh/common/AbstractSetTest.java

```diff
@@ -0,0 +1,77 @@
+package org.folio.oaipmh.common;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.List;
+
+import org.folio.rest.jaxrs.model.FilteringCondition;
+import org.folio.rest.jaxrs.model.FolioSet;
+
+import com.google.common.collect.ImmutableList;
+
+public abstract class AbstractSetTest {
+
+  protected static final String TEST_TENANT_ID = "oaiTest";
+  protected static final String TEST_USER_ID = "30fde4be-2d1a-4546-8d6c-b468caca2720";
+  protected static final String EXISTENT_SET_ID = "16287799-d37a-49fb-ac8c-09e9e9fcbd4d";
+  protected static final String NONEXISTENT_SET_ID = "a3bd69dd-d50b-4aa6-accb-c1f9abaada55";
+
+  private static FilteringCondition MOCK_FILTERING_CONDITION = new FilteringCondition().withName("fk name")
+    .withValue("fk value")
+    .withSetSpec("fk setSpec");
+  private static FilteringCondition MOCK_UPDATED_FILTERING_CONDITION = new FilteringCondition().withName("fk updated name")
+    .withValue("fk updated value")
+    .withSetSpec("fk updated setSpec");
+
+  private static List<FilteringCondition> mockFilteringConditions = ImmutableList.of(MOCK_FILTERING_CONDITION);
+  private static List<FilteringCondition> mockUpdatedFilteringConditions = ImmutableList.of(MOCK_UPDATED_FILTERING_CONDITION);
+
+  protected static FolioSet INITIAL_TEST_SET_ENTRY = new FolioSet().withId(EXISTENT_SET_ID)
+    .withName("test name")
+    .withDescription("test description")
+    .withSetSpec("test setSpec")
```

Comment:
In the schema the description for this field suggests that it actually contains a numeric value... Is that always the case?  If so, why is it a string?  Finally, should the test use realistic data?

Reply:
It's just a unit tests, there is no matter what content the models have. As well model contains text data, not numbers.

Reply:
> Generation from the last record with the same matchedId incremented by 1. Starts from 0.
In that case, this description is confusing since it suggests that the value is an integer > 0.

Reply:
![image](https://user-images.githubusercontent.com/60663044/91872727-db73f480-ec80-11ea-916d-5f865a4dc311.png)
Where do you see Integer?I can't get what you say. Could you please explain in more detail what you mean please?

Reply:
I'm taking about the description in the schema....
https://github.com/folio-org/mod-oai-pmh/blob/2bf24297a079494bd7bee483a8a01e2ec2aae73d/ramls/schemas/dto/folioSet.json#L21

Reply:
if that's not accurate please fix the description.

Reply:
"Starts at 0", "incremented by 1" suggests to me that it's an integer... 0, 1, 2, 3, ... N

Reply:
@craigmcnally, it has been fixed.
![image](https://user-images.githubusercontent.com/60663044/92006402-5acefa00-ed4d-11ea-898a-a93c282a450e.png)